Tribler/Core/SessionConfig.py

```diff
@@ -161,10 +167,16 @@ def set_permid_keypair_filename(self, keypairfilename):
         """
         self.sessconfig.set(u'general', u'eckeypairfilename', keypairfilename)
 
+    @blocking_call_on_reactor_thread
     def get_permid_keypair_filename(self):
         """ Returns the filename of the Session's keypair.
         @return An absolute path name. """
-        return self.sessconfig.get(u'general', u'eckeypairfilename')
+        file_name = self.sessconfig.get(u'general', u'eckeypairfilename')
+        if not file_name:
```

Comment:
I did that a while ago, I don't remember anymore, I guess it was getting ```None```. I will try removing that and seeing if I can reproduce whatever issue I had.